vue-frontend/locales/nl.json

```diff
@@ -42,6 +42,8 @@
     "move_stop": "Stop"
   },
   "programming": {
+    "python": "python",
+    "blockly": "blokken",
```

Comment:
I'd keep it as Blockly, as that is the programming language, maybe capitalize python as well

Reply:
Usually on primary-/highschools they refer to block and typed programmig laguages. I did add a capital (as discussed)